charts/gardener-extension-shoot-cert-service/templates/rbac-shoot-cert-service.yaml

```diff
@@ -1,38 +0,0 @@
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
```

Comment:
Definitely make sense to remove this and add it to the `Role` in `charts/internal/shoot-cert-management-seed/templates/rbac.yaml`.
However, I think we'll have a problem by doing all in one release:
1. New release is deployed with `ControllerDeployment` -> `ClusterRole` is removed
2. Running Cert-Management instances in CP namespaces don't have enough permissions because of (1)
3. Permission issue from (2) is only resolved after `extension` resource is reconciled because `Role: extensions.gardener.cloud:extension-shoot-cert-service` was updated.

We have to retain this file for at least one additional release.

Reply:
ok, file `rbac-shoot-cert-service.yaml` is back (with an comment that it is obsolete)